varya/assets/css/style-editor.css

```diff
@@ -1,285 +1,20 @@
- */
-.site-footer > *,
-.site-main > article > *,
-.site-main > .not-found > *,
-.entry-content > *,
-[class*="inner-container"] > *,
-.widget-area > * {
-	margin-top: calc( 0.666 * var(--global--spacing-vertical));
-	margin-bottom: calc( 0.666 * var(--global--spacing-vertical));
-}
-
-@media only screen and (min-width: 560px) {
-	.site-footer > *,
-	.site-main > article > *,
-	.site-main > .not-found > *,
-	.entry-content > *,
-	[class*="inner-container"] > *,
-	.widget-area > * {
-		margin-top: var(--global--spacing-vertical);
-		margin-bottom: var(--global--spacing-vertical);
-	}
-}
-
-.site-footer > *:first-child,
-.site-main > article > *:first-child,
-.site-main > .not-found > *:first-child,
-.entry-content > *:first-child,
-[class*="inner-container"] > *:first-child,
-.widget-area > *:first-child {
-	margin-top: 0;
-}
-
-.site-footer > *:last-child,
-.site-main > article > *:last-child,
-.site-main > .not-found > *:last-child,
-.entry-content > *:last-child,
-[class*="inner-container"] > *:last-child,
-.widget-area > *:last-child {
-	margin-bottom: 0;
-}
-
-/*
- * Block & non-gutenberg content wrapper children
- * - Sets spacing-unit margins
- */
-.site-header > *,
-.entry-header > *,
-.post-thumbnail > *,
-.page-content > *,
-.comment-content > *,
-.author-bio > *,
-.widget-area > .widget > * {
-	margin-top: var(--global--spacing-unit);
-	margin-bottom: var(--global--spacing-unit);
-}
-
-.site-header > *:first-child,
-.entry-header > *:first-child,
-.post-thumbnail > *:first-child,
-.page-content > *:first-child,
-.comment-content > *:first-child,
-.author-bio > *:first-child,
-.widget-area > .widget > *:first-child {
-	margin-top: 0;
-}
-
-.site-header > *:last-child,
-.entry-header > *:last-child,
-.post-thumbnail > *:last-child,
-.page-content > *:last-child,
-.comment-content > *:last-child,
-.author-bio > *:last-child,
-.widget-area > .widget > *:last-child {
-	margin-bottom: 0;
-}
-
-/*
- * .entry-content children specific controls
- * - Adds special margin overrides for alignment utility classes
- */
-.entry-content > * {
-	/* Reset alignleft and alignright margins after alignfull */
-}
-
-.entry-content > *.alignleft, .entry-content > *.alignright,
-.entry-content > *.alignleft:first-child + *,
-.entry-content > *.alignright:first-child + *, .entry-content > *.alignfull {
-	margin-top: 0;
-}
-
-.entry-content > *:last-child, .entry-content > *.alignfull {
-	margin-bottom: 0;
-}
-
-.entry-content > *.alignfull + .alignleft,
-.entry-content > *.alignfull + .alignright {
-	margin-top: var(--global--spacing-vertical);
-}
-
 body {
 	color: var(--global--color-foreground);
 	background-color: var(--global--color-background);
 	font-family: var(--global--font-secondary);
 	font-size: var(--global--font-size-root);
 	font-weight: normal;
-	line-height: var(--global--font-line-height-body);
 	-moz-osx-font-smoothing: grayscale;
 	-webkit-font-smoothing: antialiased;
 }
 
+p {
+	line-height: var(--global--line-height-body);
```

Comment:
I'm not sure this is actually necessary anymore – I tried turning this off in the inspector and things looked the same as they did with it enabled.

Reply:
Also a sidenote: this line height value should be `1.7`, not `1.78`. Would you mind updating that in this PR since you're working on line height anyway?

Reply:
> I'm not sure this is actually necessary anymore – I tried turning this off in the inspector and things looked the same as they did with it enabled.

Yes but without it, if a child-theme has a different line-height, than they won’t be able to change it. 

> This line height value should be 1.7, not 1.78. 

Yup, will do. Afterward check to see that you you notice a change when enabling/disabling `line-height: var(--global--line-height-body);` like you did above—you should see a subtle change there :-)